welcome/src/main/kotlin/edu/artic/welcome/WelcomeEventsAdapter.kt

```diff
@@ -13,12 +15,21 @@ import kotlinx.android.synthetic.main.welcome_event_cell_layout.view.*
  * @author Sameer Dhakal (Fuzz)
  */
 
-class WelcomeEventsAdapter : AutoHolderRecyclerViewAdapter<WelcomeEventCellViewModel>() {
+class WelcomeEventsAdapter(private val viewModel: WelcomeViewModel) : AutoHolderRecyclerViewAdapter<WelcomeEventCellViewModel>() {
     override fun View.onBindView(item: WelcomeEventCellViewModel, position: Int) {
+        clicks()
+                .defaultThrottle()
+                .subscribe { viewModel.onClickEvent(position, item.event) }
+                .disposedBy(item.viewDisposeBag)
+
         item.eventTitle
```

Comment:
im worried these bindings will compact as you scroll up and and down a list. itll keep adding new bindings that do not get disposed.

Reply:
done